static/scripts/util.js

```diff
@@ -105,4 +105,15 @@ $(document).on('click', '.remove-tag', function (e) {
     $(this).parent('.badge').remove();
 });
 
+$(document).on('click', '#filterIcon', function (e) {
+    e.preventDefault();
+    $('#filterRow').toggle();
+});
 
+$(document).on('keyup', '#filterRow input', function () {
+    let column = $(this).data('column');
+    let value = $(this).val().toLowerCase();
+    $('table tbody tr').filter(function () {
+        $(this).toggle($(this).find('td').eq(column).text().toLowerCase().indexOf(value) > -1);
```

Comment:
Please add a little comment here, or split the line into multiple ones, to better understand you fetch the `column`-th column content, set it to lower case and check if it contains the searched value, then only toogle the column if this is the case.